tests/dem/insertion_uniform.cc

```diff
@@ -58,26 +58,25 @@ test()
   DEMSolverParameters<dim> dem_parameters;
 
   // Defining simulation general parameters
-  const unsigned int n_properties                 = 21;
-  dem_parameters.insertionInfo.x_min              = -0.05;
-  dem_parameters.insertionInfo.y_min              = -0.05;
-  dem_parameters.insertionInfo.z_min              = -0.05;
-  dem_parameters.insertionInfo.x_max              = 0.05;
-  dem_parameters.insertionInfo.y_max              = 0.05;
-  dem_parameters.insertionInfo.z_max              = 0.05;
-  dem_parameters.insertionInfo.inserted_this_step = 10;
-  dem_parameters.insertionInfo.distance_threshold = 2;
-  dem_parameters.physicalProperties.diameter      = 0.005;
-  dem_parameters.physicalProperties.density       = 2500;
+  const unsigned int n_properties                        = 21;
+  dem_parameters.insertionInfo.x_min                     = -0.05;
+  dem_parameters.insertionInfo.y_min                     = -0.05;
+  dem_parameters.insertionInfo.z_min                     = -0.05;
+  dem_parameters.insertionInfo.x_max                     = 0.05;
+  dem_parameters.insertionInfo.y_max                     = 0.05;
+  dem_parameters.insertionInfo.z_max                     = 0.05;
+  dem_parameters.insertionInfo.inserted_this_step        = 10;
+  dem_parameters.insertionInfo.distance_threshold        = 2;
+  dem_parameters.physicalProperties.diameter             = 0.005;
+  dem_parameters.physicalProperties.density              = 2500;
+  dem_parameters.simulationControl.total_particle_number = 10;
```

Comment:
I do not understand why the total particle number is still in the simulation control. It should be in the insertion parameters...